packages/ag-charts-community/src/chart/series/cartesian/lineSeries.test.ts

```diff
@@ -447,4 +447,90 @@ describe('LineSeries', () => {
             }
         });
     });
+
+    describe('data per series', () => {
+        it('with category keys should render as expected', async () => {
+            const options: AgChartOptions = {
+                series: [
+                    {
+                        data: [
+                            { key: '2020', value: 300 },
+                            { key: '2021', value: 200 },
+                            { key: '2022', value: 350 },
+                            { key: '2023', value: 400 },
+                        ],
+                        type: 'line',
+                        xKey: 'key',
+                        yKey: 'value',
+                    },
+                    {
+                        data: [
+                            { key: '2022', value: 100 },
+                            { key: '2023', value: 130 },
+                            { key: '2024', value: 160 },
+                            { key: '2025', value: 200 },
+                        ],
+                        type: 'line',
+                        xKey: 'key',
+                        yKey: 'value',
+                    },
+                ],
+            };
+
+            prepareTestOptions(options);
+
+            chart = AgCharts.create(options) as Chart;
+
+            // TODO: replace with `compare()` with 0 percent threshold
+            await waitForChartStability(chart);
+
+            const imageData = extractImageData(ctx);
+            expect(imageData).toMatchImageSnapshot({
+                failureThreshold: 0,
+                failureThresholdType: 'percent',
+            });
```

Comment:
These tests didn't fail with the change reverted and using the default threshold. But changing the threshold for all the tests led to them all updating (just the removal of the markers so probably fine, but worth saving for another PR).